themes/default/content/docs/guides/organizing-projects-stacks/_index.md

```diff
@@ -158,13 +205,64 @@ Back to our example, our service needs a database and a subnet (or other network
 
 These component resources can be packaged up and stored alongside all of your other package management, so consumers in your organization can access them like any other library or package. If we want to add component resources to our monorepo example, it will look like this:
 
+{{< chooser language "typescript,go" / >}}
+
+{{% choosable language typescript %}}
+
+```
+├── infrastructure
+│   ├── index.ts
+│   ├── Pulumi.yaml
+│   ├── Pulumi.dev.yaml
+│   └── Pulumi.prod.yaml
+├── myApp
+│   ├── index.ts
+│   ├── Pulumi.yaml
+│   ├── Pulumi.dev.yaml
+│   ├── Pulumi.staging.yaml
+│   └── Pulumi.prod.yaml
+├── pkg
+│   └──application
+│     └── app.ts
+└── .etc
+```
+
+{{% /choosable %}}
+
+{{% choosable language go %}}
+
 ```
 ├── infrastructure
 │   ├── main.go
+│   ├── Pulumi.yaml
 │   ├── Pulumi.dev.yaml
 │   └── Pulumi.prod.yaml
 ├── myApp
 │   ├── main.go
+│   ├── Pulumi.yaml
+│   ├── Pulumi.dev.yaml
+│   ├── Pulumi.staging.yaml
+│   └── Pulumi.prod.yaml
+├── pkg
+│   └──application
+│     └── app.go
+└── .etc
+```
+
+{{% /choosable %}}
+
+To be clear, each of the applications/services inside our monorepo (and the `infrastructure` project is treated the same way as each independent application/service) are a separate Pulumi project, with their own stacks, and their own `Pulumi.yaml`. Given that each service is a separate Pulumi project, they can all use different programming languages. Let's take a look at how it might look if the `infrastructure` team prefers to write in Go, and the myApp team prefers TypeScript:
```

Comment:
the parenthetical after monorepo, makes this a bit difficult to read. can we include it in a later sentence instead?